gui/app/controls/ScopeView.qml

```diff
@@ -66,103 +66,18 @@ Item {
         }
     }
 
-    ChartView
-    {
-        id: chartView
-
-        //due to a bug on Qt charts 5.11, which is used at the moment
-        // in Raspiberry OS, plotArea property is marked as read-only
-        // making it impossible to control the drawing area.
-        // This was fixed in Qt 5.13 (maybe even on 5.12). Once we have
-        // the fix we can uncomment the following code as the proper
-        // solution:
-
-//         anchors {
-//            top: parent.top; topMargin: 0
-//            left: parent.left; leftMargin: 48
-//            right: parent.right; rightMargin:8
-//            bottom: parent.bottom; bottomMargin: 8
-//        }
-        //plotArea: Qt.rect(0, 0, chartView.width, chartView.height)
-
-        //meanwhile we have to do guess work and some magic numbers to
-        //increase the size of chartview beyond its parent in order to
-        //get a proper rectangle for the plotArea
-
+    PeriodicGraph {
```

Comment:
"Periodic" sounds misleading - the graph of ventilator readings is not periodic in the sense of https://en.wikipedia.org/wiki/Periodic_function.